dev/styles/main/plugins/search/search.less

```diff
@@ -167,3 +167,22 @@
     background: darken(@main-body-bg, 10%);
     color: @main-text-color;
 }
+
+input[type="search"] {
```

Comment:
Should you create a parametric mixin which can be later used with other form field placeholders?

see: http://lesscss.org/features/#mixins-parametric-feature